app/src/main/java/com/safetyheads/akademiaandroida/contact_with_us/ContactWithUsFragment.kt

```diff
@@ -0,0 +1,174 @@
+package com.safetyheads.akademiaandroida.contact_with_us
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.safetyheads.akademiaandroida.databinding.FragmentContactWithUsBinding
+
+class ContactWithUsFragment : Fragment() {
+    companion object {
+        val TAG = "ContactWithUsFragment"
+
+        val Facebook = "Facebook"
+        val Instagram = "Instagram"
+        val Linkedin = "Linkedin"
+        val YouTube = "YouTube"
```

Comment:
Enum or seald class will be more sutable here and then you could reduce amount of code lines in this class.